control-plane/msp-operator/src/main.rs

```diff
@@ -435,93 +435,101 @@ impl ResourceContext {
             // we updated the resource as an error stop reconciliation
             return Err(Error::ReconcileError { name: self.name() });
         }
-        match self
+        let check_if_block_device = if self.spec.disks[0].starts_with("aio:///") {
+            let path = normalize_disk(&self.spec.disks[0][6..]);
+            !std::path::Path::new(&path).exists()
+        } else {
+            false
+        };
+        if check_if_block_device {
```

Comment:
If you dont check if the block device exists (for whatever reason) then the only way to ever understand what's wrong here is to look in the logs of the node which was selected and read its error logs to understand what was wrong. IMHO the RPC methods in the mayastor node itself need to be extended to report files using AIO.

Reply:
I understand that this makes sense, but it is a non-trivial change, and I highly doubt that my Rust skills are appropriate to take this on. :(

Would a different approach be acceptable, by which I mean a configuration option to disable validation on the side of `msp-operator`?

Reply:
Yeah as an interim solution I think an option to disable validation, would be appropriate, as long as by default we do validate.